src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.scss

```diff
@@ -9,6 +9,9 @@
     > div{
         padding: 0;
     }
+    #wi-comment-add-comment {
+        height: em(64);
```

Comment:
We need scroll on the almEditable to support multi-line comments.
On demo.almighty
![ondemo](https://cloud.githubusercontent.com/assets/1892722/22051844/212ad96c-dd6c-11e6-94d5-d3d6c6c22e86.png)

Your PR
![commentsoverflow](https://cloud.githubusercontent.com/assets/1892722/22051853/2b1c2e4e-dd6c-11e6-8fb5-e50898712163.png)